packages/website/pages/account/payment.js

```diff
@@ -187,6 +197,9 @@ const PaymentSettingsPage = props => {
           onClose={() => {
             setIsPaymentPlanModalOpen(false);
             setHasAcceptedTerms(false);
+            if (planQueryParam) {
+              history.pushState({}, '', window.location.href.replace(/plan=[^&]+&?/, ''));
```

Comment:
Is this inteded to wipe the query param?

If so, I think it's kinda nice to keep it around. That way the user can share the URL to that tier to themselves or others (e.g. via user-agent share functionality, copy url bar and paste elsewhere, etc)

Just curious, what is the motivation of the pushState and, if I'm reading it right, clearing the query param? If it's not strictly needed we might as well not do it.

Reply:
so, if the user closes the modal, it's their intention _not_ to see it again.  if they refresh the browser after closing the modal and then see the modal again, i figured it would be a little annoying.  if they click the browser back button they'll see the modal and query param again.  lmk what you want.

Reply:
ah, it's onclose. Nice. Sry for missing that.

Reply:
>  if they click the browser back button they'll see the modal and query param again with my current implementation.

awesome